CODEOWNERS

```diff
@@ -204,6 +204,8 @@
 * @cilium/tophat
```

Comment:
Not related to this PR, but it looks like the CODEOWNERS file is invalid on this branch. Don't we usually replace the whole codeowners with a more minimal set of owners on older branches...?